### PR TITLE
US152268 - Migrate card vdiff tests

### PR DESCRIPTION
**Pull request by svanherk**
Mostly basic migration now that `vdiff-include` exists. Will make some notes inline.

Final Report: https://vdiff.d2l.dev/BrightspaceUI/core/c860cefe9b1172ab232848d6a05db8789902645a/2023-09-06_14-37-22/report/

### CODE REVIEW

File: components/card/test/card.vdiff.js [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	const { alignCenter, content, subtle } =  { alignCenter: false, subtle: false, ...opts };
+	return html`
+		<d2l-card style="height: 260px; width: 210px;" ?align-center="${alignCenter}" ?subtle="${subtle}">
+			${content}
+		</d2l-card>
+	`;
+}
+function createLinkCardTemplate(opts) {
+	return html`
+		<d2l-card style="height: 260px; width: 210px;" text="Link Text" href="javascript:void(0);">
```
Comment (svanherk) [inline components/card/test/card.vdiff.js:19]: Setting an actual `href` here caused the test to navigate.

File: components/card/card.js
[the remark this answers is not in the document]
Comment (dlockhart) [reply components/card/card.js:1]: Another option here would be to use the `await nextFrame()` helper from `@brightspace-ui/testing`. They should be equivalent though.

File: components/card/test/card-footer-link.vdiff.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import '../../tooltip/tooltip.js';
+import '../card-footer-link.js';
+import { expect, fixture, focusElem, html, oneEvent } from '@brightspace-ui/testing';
+
+describe('card-footer-link', () => {
+	[
+		{ name: 'no-secondary', template: html`
+			<d2l-card-footer-link id="no-secondary" icon="tier1:gear" text="Settings" href="https://www.d2l.com">
+				<d2l-tooltip class="vdiff-include" slot="tooltip" for="no-secondary">No new settings</d2l-tooltip>
```
[the remark this answers is not in the document]
Comment (dlockhart) [reply components/card/test/card-footer-link.vdiff.js:9]: I think it's best the way you have it even if that means a change to the goldens.

**Review by dlockhart — APPROVED** [review]

File: components/card/card.js
[the remark this answers is not in the document]
Comment (svanherk) [reply components/card/card.js:1]: In `card` itself?

[the remark this answers is not in the document]
Comment (dlockhart) [reply components/card/card.js:1]: Ah sorry forgot where this was -- not in card, this is good how it is!

**Pull request merged**